SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+## Supported Projects
+
+Please follow this guidance when reporting security issues affecting:
+
+- [Shields.io](https://shields.io)
+- [Raster.shields.io](https://easter.shields.io)
```

Comment:
Probably doesn't really matter (since we're redirecting on the root path) but:
```suggestion
- [Raster.shields.io](https://raster.shields.io)
```

Reply:
Lol. I left a little easter egg for you there :laughing:

Reply:
hahaha. well played!